traffic_ops/traffic_ops_golang/server/servers.go

```diff
@@ -782,6 +660,11 @@ func validateV3(s *tc.ServerV30, tx *sql.Tx) (string, error) {
 	if !serviceAddrV6Found && !serviceAddrV4Found {
 		errs = append(errs, errors.New("a server must have at least one service address"))
 	}
+
+	if s.UpdPending == nil {
+		errs = append(errs, errors.New("'updPending' cannot be null"))
```

Comment:
Is this true? I'd think we wouldn't really care about this field for creating or updating a server... Plus, this might be a breaking change if this validation wasn't already being enforced somewhere else.

Reply:
See previous comment.